extensions/mssql/src/objectExplorerNodeProvider/fileSources.ts

```diff
@@ -313,11 +222,11 @@ export class HdfsFileSource implements IFileSource {
 				}
 			})
 			.on('error', (err) => {
-				errorMsg = utils.getErrorMessage(err);
-				reject(errorMsg);
+				error = <HdfsError>err;
+				reject(error);
```

Comment:
`<HdfsError>` does not do anything, but preventing syntax error.
The `err` object is `HdfsError` type originally because I attached error handler to `fs.ReadStream` that emits `HdfsError` error object (the return value of `this.parseError(response, data.toString())`):
```
...
req.on('response', (response) => {
    // Handle remote exceptions
    // Remove all data handlers and parse error data
    if (this.isError(response)) {
        req.removeAllListeners('data');
        req.on('data', (data) => {
            req.emit('error', this.parseError(response, data.toString()));
            req.end();
        });
    } 
...
```
I already tested and confirmed the `err` is in `HdfsError` type.